frontend/src/lib/variables.js

```diff
@@ -1,6 +1,7 @@
 export const PYTHON_API = import.meta.env.VITE_API_PATH
 export const GO_API = import.meta.env.VITE_GO_API_PATH
 
+export const PYTHON_STATIC = `${PYTHON_API}/static`
```

Comment:
I think we should follow the logic of not mentioning the language, so we can hot-swap the host(like the IMG-variables)
```suggestion
export const STATIC = `${PYTHON_API}/static`
```

Reply:
Done